homeassistant/components/homekit/type_sensors.py

```diff
@@ -141,16 +149,22 @@ def __init__(self, *args):
         """Initialize a AirQualitySensor accessory object."""
         super().__init__(*args, category=CATEGORY_SENSOR)
         state = self.hass.states.get(self.entity_id)
+
+        self.init()
```

Comment:
It looks like `AirQualitySensor` is no longer used?

Reply:
I think the AirQualitySensor can be treated as a generic class for non PM sensor. User can add any non PM 2.5 and PM10 air sensor to the homekit by using this class.